modules/ROOT/pages/questions/question-02.adoc

```diff
@@ -0,0 +1,8 @@
+2. I’ve tentatively noted each ‘Plugin moved to Premium’ in the
+   *Migration Guide* with an admonishment that
+
+   *If you are 1) using the Free version of {productname} and 2) rely
+   on the `{plugin-name}` plugin, upgrading to {productname} 6.0 is
+   not recommended.*
+
+   Is this correct?
```

Comment:
@EkimChau

Reply:
It's definitely not correct. We will always recommend upgrading to the latest release, these plugins are still available just not for free.